models/FeaturedItemsSection.yaml

```diff
@@ -0,0 +1,121 @@
+type: object
```

Comment:
@TomasBankauskas Can you please review models, I'm not up to date with the latest model values, styles etc.